Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN /bin/echo -n "Running on " && /usr/bin/arch
 COPY . /opt/openhabian/
 COPY openhabian.conf.dist /etc/openhabian.conf
 RUN sed -i 's#repositoryurl=https://github.com/openhab/openhabian.git#repositoryurl=https://github.com/mstormi/openhabian.git#' /etc/openhabian.conf
-RUN sed -i 's#clonebranch=master#clonebranch=buildfixes#' /etc/openhabian.conf
+RUN sed -i 's#clonebranch=master#clonebranch=buildsystem-bats#' /etc/openhabian.conf
```

Comment:
Hey @mstormi , I think I asked this already today but cannot find it right now.
What is the strategy how we get rid of hard coding the branch name here? Is this adressed in the following PRs?

Reply:
No you didn't ask yet. Can discuss in team board if you like.
Essentially this is the codepiece where it's injected
If you can come up with an idea how to retrieve the repo name in Travis (env variables maybe ?) we might auto-code that but for the time being it is hardcoded in the PR's Dockerfile

Reply:
TRAVIS_PULL_REQUEST_BRANCH could be the thing. Just have a look at https://docs.travis-ci.com/user/environment-variables/#convenience-variables . 
Maybe also TRAVIS_PULL_REQUEST_SLUG is useful for tests on your own repo, before you do the pull req....

Reply:
;-) right just found and tested those, too ... yes they're set in .travis.yml will build that into  PR #683